### PR TITLE
Memory viewer in the debugger

### PR DESCRIPTION
**Pull request by remicalixte**
ram and vram
added back the pause button

### CODE REVIEW

File: debugger/static/memory.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    static tagName() {
+        return 'memory-div';
+    }
+
+    constructor() {
+        super();
+
+        this.id = "memory";
+
+        this.ram = document.createElement("pre");
```
Comment (sfluor) [inline debugger/static/memory.js:11]: nit: should we add two headers to indicate that the first part is the RAM and the second one the VRAM ?

**Review by sfluor — APPROVED** [review]
:ok_hand: left one comment

File: debugger/static/memory.js (hunk 1 above)
Comment (remicalixte) [reply debugger/static/memory.js:11]: Yes right. do you prefer both tabs side by side or one below the other like it is now ?

**Comment by remicalixte** [comment]
Looks like this now 
image

**Review by sfluor — APPROVED** [review]

**Pull request merged**